### DISCUSSION TITLE
第三方軟體支援離線播放yt、阻擋廣告、背景播放

### DISCUSSION THREAD

**Discussion by PoooYen** [1101-GS4719-C/Python-Community-GS4719-C#499] [General]
這有點類似手機版的ad blocker，但又可支援離線播放真滴香，缺點是不能適用在蘋果手機  :  )，安卓系統的人不仿試試類似這種yt preium的這種軟體ㄅ，但是個資上的提供還是需要小心，畢竟是第三方的app。

https://www.kocpc.com.tw/archives/432135

**Comment by 11timyyy** [comment]
好像有些能在安卓上下載的東西蘋果都不能下載🥲

**Comment by chupochun** [comment]
太好了這樣以後看影片就不用等廣告了

**Comment by hapiwangy** [comment]
蘋果用戶+1，但其實有時候看看廣靠還是挺有趣的。並且我感覺看廣告像是使用者付費，所以看一下好像也沒什麼關係

**Comment by leon10456** [comment]
vpn去印度 一個月十幾塊的yt premium 不香嗎 重點是還能在各種平台使用

**Comment by henrywen547** [comment]
用蘋果的如果是用youtube聽音樂不想要廣告的話是可以用這個方法(很安全ㄉ
[YouTube 背景播放、鎖屏播放音樂 SOP！免裝 App，9 步驟就能成功](https://www.managertoday.com.tw/articles/view/58761?)
但我知道這個方法之前已經有youtubemusic Premium家庭號了哭啊

**Comment by dayu1004** [comment]
YT的廣告已經猖狂到沒有用ad block的時候看影片會被廣告洗到生氣了哈哈哈 謝謝分享！

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，以後就不會因為廣告而影響影片觀看。
